test/cycles/Makefile

```diff
@@ -14,8 +14,8 @@ $(ROOT)/cycle_abort.done: RETCODE=1
 $(ROOT)/cycle_abort.done: LINES=7
 $(ROOT)/cycle_print.done: RETCODE=0
 $(ROOT)/cycle_print.done: LINES=6
-$(ROOT)/cycle_deprecate.done: RETCODE=0
-$(ROOT)/cycle_deprecate.done: LINES=4
+$(ROOT)/cycle_deprecate.done: RETCODE=1
```

Comment:
Shouldn't this be deleted? It doesn't seem to reflect the reality

Reply:
It does, it's the same as abort now + 1 line for the message that the option is now removed.

Reply:
I was forgetting the fact that there's still a message being shown. Makes sense